tls-client/main.cpp

```diff
@@ -406,12 +409,17 @@ class HelloHTTPS {
     mbedtls_x509_crt _cacert;
     mbedtls_ssl_context _ssl;
     mbedtls_ssl_config _ssl_conf;
+    mbedtls_platform_context* _platform_ctx;
 };
 
 /**
  * The main loop of the HTTPS Hello World test
  */
 int main() {
+    mbedtls_platform_context platform_ctx;
+    if(mbedtls_platform_setup(&platform_ctx) != 0) {
+        return -1;
```

Comment:
I suggest not covering up the return code here and also adding something like:
```
printf( "Platform initialization failed with error -%04X",  -ret );
```
Not sure what the format should be but thats how we normally print error codes.